### PR TITLE
Changes to jk/redeemables

### PR DESCRIPTION
**Pull request by p-woznikowski**
- Reviewed descriptions.
- Added a new `FilterConditionsDateTime` schema that works better with `created_at` filters.
  - It also includes `$more_than` and `$less_than` conditions.
  - I think the previous schema for `created_at` (`FilterConditionsString`) was misleading because even if `$is`, `$is_not`, `$in`, `$not_in` work in some way, they didn't work with what the documentation generated, so I propose removing these altogether and provide the `FilterConditionsDateTime`.
- Added the `campaign_id` filter to both endpoints, because it works for `List Customer's Redeemables` and it's mentioned in [Listing referral holders](https://github.com/rspective/voucherify-mono/pull/17462#issue-2192273659).
- Changed paging for the `customer` Markdown files.
- Added a couple parameters
- Added `has_more` and `more_starting_after` fields.
- Added `BETA` info to the endpoints.
- Changed `Get Member Holders` to `List Member Holders`.
- Added info about `User permissions` to the three endpoints.
- Changed in some places the info about `in ISO 8601 format.` because strictly speaking, it's grammatically nonsensical the way it's used throughout the docs. "...when the redeemable was created in ISO 8601 format` suggests that the redeemable was created in the ISO format, not the timestamp.
- Added relevant Markdown files for the endpoints.

### CODE REVIEW

**Review by jkaliszuk — APPROVED** [review]

File: reference/OpenAPI.json [hunk 1]
```diff
@@ -27451,11 +27533,14 @@
...
           "campaign_id": {
             "type": "string",
-            "description": "Unique campaign ID, assigned by Voucherify.",
+            "nullable": true,
+            "description": "Unique campaign identifier, assigned by Voucherify.",
             "example": "camp_weer1c3p5ZgktqfW56RfoNaG"
           },
           "campaign_type": {
             "type": "string",
+            "nullable": true,
```
Comment (p-zielinski) [inline reference/OpenAPI.json:27542]: kiedy dostajemy null-a?

Comment (p-zielinski) [reply reference/OpenAPI.json:27542]: daje DNM do czasu odp

Comment (p-woznikowski) [reply reference/OpenAPI.json:27542]: `campaign_id` i `campaign_type` dostają `null`, gdy przypisany "redeemable" jest to kod standalone.

**Comment by p-zielinski** [comment]
Z tego co widzę to jest już na mainie

**Pull request closed without merging**